manticore/native/cpu/x86.py

```diff
@@ -2079,7 +2079,7 @@ def NEG(cpu, dest):
         :param dest: destination operand.
         """
         source = dest.read()
-        res = dest.write(-source)
+        res = dest.write((~source) + 1)
```

Comment:
I'd like for someone to confirm that this change is necessary/correct.

It seems very odd that this change is required to get tests to pass 🤔